### DISCUSSION TITLE
How to add a counter column in Tbl_MasterColumns

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#5701] [Help]
How to add a counter column in Tbl_MasterColumns

**Comment by ra-bedo11** [comment]
Please HelpMe

**Reply by minhhungit** [reply]
Hello @ra-bedo11 

```javascript
What is the fu.k Tbl_MasterColumns  and what kind of counter you want to implement ?
```

That is what somebody else will ask you, but I will ask you can you **please** explain more detail about exactly how do you want to implement the counter, I am not sure what you want to count, where you will place the counter, and what does it look like

So if you can explain your requirement more detail it will help us a lot and I trust that you will be helped soon
